relayer/src/config.ts

```diff
@@ -0,0 +1,64 @@
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+type Config = {
+  accountSeed: string;
+  targetChainUrl: string;
+  sourceChainUrls: string[];
+};
```

Comment:
This should have been an interface instead of type alias. Never actually seen this style.

In future updates would be nice to have this as a class with public readonly properties instead, those properties don't need to be mutable (and TypeScript allows to make them such), but will work for now.